spec/conformance/server.rb

```diff
@@ -60,8 +60,15 @@ def descriptor
     client_data_json: client_data_json
   )
 
+  public_key_credential = WebAuthn::PublicKeyCredential.new(
```

Comment:
Still considering shortening this to `WebAuthn::Credential`.

Wanted to match exactly the name that the spec uses (https://www.w3.org/TR/webauthn/#iface-pkcredential), but at the same time it feels unnecessarily long...